met-web/src/routes/LandingPage/LandingPageElements.tsx

```diff
@@ -3,5 +3,7 @@ import { Row, Column } from '../../components/common';
 
 export const SearchContainer = styled(Row)`
```

Comment:
we don't need to build our own search container. We can use material UI's search bar. Check https://mui.com/material-ui/react-text-field/ the Customization section